foundry/chatBubbles.d.ts

```diff
@@ -1,16 +1,26 @@
-// @TODO:
-
 /**
  * The Chat Bubble Class
  * This application displays a temporary message sent from a particular Token in the active Scene.
  * The message is displayed on the HUD layer just above the Token.
  */
 declare class ChatBubbles {
+  /**
+   * @defaultValue `'templates/hud/chat-bubble.html'`
+   */
+  template: string;
+
   /**
    * Track active Chat Bubbles
+   * @defaultValue `{}`
    */
   bubbles: object;
 
+  /**
+   * Track which Token was most recently panned to highlight
+   * Use this to avoid repeat panning
+   */
+  protected _panned: Token | null;
+
   constructor();
```

Comment:
The constructor should be at the top. Please follow the order of `foundry.js` strictly.